runner/unitTestRunner.go

```diff
@@ -1,55 +0,0 @@
-package runner
-
-import (
-	"encoding/json"
-	"encoding/xml"
-	"io/ioutil"
-	"log"
-	"net/http"
-)
-
-// UnitTestResult holds the unit test result created by junit
-type UnitTestResult struct {
-	Tests    string   `xml:"tests,attr"`
-	Failures string   `xml:"failures,attr"`
-	Errors   string   `xml:"errors,attr"`
-	TestCase TestCase `xml:"testcase"`
-}
-
-// TestCase holds a test case created by junit
-type TestCase struct {
-	Name    string  `xml:"name,attr"`
-	Time    string  `xml:"time,attr"`
-	Failure Failure `xml:"failure"`
-}
-
-// Failure holds a failure created by junit
-type Failure struct {
-	Message string `xml:"message,attr"`
-}
-
-const testResultsFileName = "/build/test-results/TEST-com.coduno.TestApplication.xml"
```

Comment:
I would expect that line just to move somewhere, but it's gone completely. Please explain.